src/vidconv/vconv.c

```diff
@@ -180,6 +180,21 @@ static inline void yuv2rgb555(uint8_t *rgb, uint8_t y,
 }
 
 
+static inline void _yuv2rgb(uint8_t *rgb, uint8_t y, uint8_t u, uint8_t v)
```

Comment:
we already have a yuv2rgb() function, similar to this. Let's reuse that instead.